docs/advanced/urgency.md

```diff
@@ -120,3 +120,7 @@ urgency = 0.0 + 6.0 + 5.0 + 0.0 = 11.0
 A task that has no due date, a "high" priority, is scheduled for tomorrow, and starts tomorrow:
 urgency = 0.0 + 6.0 + 0.0 - 3.0 = 3.0
 ```
+
+## How to Display the Urgency score
```

Comment:
Nice. I think it's worth adding a table-of-contents to this page now, so people can see at a glance that it is possible to display the scores.

The Layout page shows how to do this.

Reply:
Enabled the toc now.

Reply:
What you did was logical, but did not change the rendered page:

![image](https://user-images.githubusercontent.com/4840096/190888607-127fb474-c46e-441c-973a-20748f7a0f23.png)

I've taken the liberty of pushing a change to the required lines - for info, they are:

```text
{: .no_toc }

<details open markdown="block">
  <summary>
    Table of contents
  </summary>
  {: .text-delta }
1. TOC
{:toc}
</details>
```